package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "6.9.0",
+  "version": "6.9.1",
```

Comment:
This is a significant change (new Legal Names API) -- change version to 6.10.0.

Also, at the top of the PR, change it to "commmit into" the Legal Name feature branch.